### PR TITLE
SOC2 - CI-86 - monitoring and alerting for MIs

### PR DESCRIPTION
**Pull request by filiphaftek**
[issue](https://github.com/sourcegraph/security-issues/issues/254)

Description of monitoring and alerting for managed instances - same for v1.0 and v1.1

### CODE REVIEW

File: content/departments/product-engineering/engineering/cloud/devops/managed/index.md [hunk 1]
```diff
@@ -102,6 +103,30 @@ The main limitation of this model is that an underlying GCP infrastructure outag
...
+1. When alert is triggered, it is sent to Opsgenie channel:
+
+- [uptime check channel](https://github.com/sourcegraph/deploy-sourcegraph-managed/blob/main/modules/terraform-managed-instance-new/infrastructure.tf#L557)
+- [metrics monitoring channel](https://github.com/sourcegraph/deploy-sourcegraph-managed/blob/main/monitoring/alerting.tf#L24)
+
+2. From Opsgenie, alert is sent to [on-call DevOps](../index.md#on-call) and Slack channels (#opsgenie, #cloud-devops).
+
+3. On-call DevOps has to decide, what is the alert type and if [incident](../../../../engineering/process/incidents/index.md) should be opened and follow the procedure to perform the incident. On-call DevOps should use [managed instances operations](./operations.md) to check, assess and repair broken managed instance.
+
+4. When alert is closed via incident resolution, post-mortem actions has to be assigned and performed.
```
Comment (sanderginn) [inline content/departments/product-engineering/engineering/cloud/devops/managed/index.md]: How do we maintain post-mortems? Do we use Opsgenie for it or do we have a Google Drive directory for it?

**Comment by sanderginn** [comment]
> how to we track the individual steps during such an incident including its resolution and postmortem

Maybe we should include that we open a slack channel with an incident ID prefixed in the name where we communicate about the resolution?

**Comment by sanderginn** [comment]
I think there could be some more info on 
> how the monitoring is setup

Linking to [the VM level](https://github.com/sourcegraph/deploy-sourcegraph-managed/tree/main/monitoring) TF config and [the service level](https://github.com/sourcegraph/sourcegraph/tree/main/monitoring/definitions) monitoring definitions should be sufficient I think

File: content/departments/product-engineering/engineering/cloud/devops/managed/index.md (hunk 1 above)
Comment (filiphaftek) [reply content/departments/product-engineering/engineering/cloud/devops/managed/index.md]: Thx @sanderginn ! I have added links to post-mortem and process for incidents in handbook - I would not copy-paste it here, but rather use the links if that's ok for you?

**Comment by filiphaftek** [comment]
> > how to we track the individual steps during such an incident including its resolution and postmortem
> 
> Maybe we should include that we open a slack channel with an incident ID prefixed in the name where we communicate about the resolution?

I have added links to Incident Process, as we have all described there (do not like to duplicate the handbook parts) - is it enough?

**Comment by filiphaftek** [comment]
> I think there could be some more info on
> 
> > how the monitoring is setup
> 
> Linking to [the VM level](https://github.com/sourcegraph/deploy-sourcegraph-managed/tree/main/monitoring) TF config and [the service level](https://github.com/sourcegraph/sourcegraph/tree/main/monitoring/definitions) monitoring definitions should be sufficient I think

Added scope project monitoring description and link. For VM level monitoring, we have only uptime check + alert, and this is included already, so I hope it's sufficient?

**Comment by sanderginn** [comment]
> Added scope project monitoring description and link. For VM level monitoring, we have only uptime check + alert, and this is included already, so I hope it's sufficient?

What about the new dashboard? I think that's worthy of mentioning, it's really helpful in my opinion

File: content/departments/product-engineering/engineering/cloud/devops/managed/index.md (hunk 1 above)
Comment (sanderginn) [reply content/departments/product-engineering/engineering/cloud/devops/managed/index.md]: :D I didn't open the incident link. That did describe everything I commented on. But you made it more explicit so 👍

**Review by sanderginn — APPROVED** [review]
I think it's good 👍

**Review by michaellzc — APPROVED** [review]

**Pull request merged**